cfgov/jinja2/v1/es/es-base.html

```diff
@@ -16,11 +16,6 @@
     <meta property="fb:page_id" content="141576752553390">
 {% endblock %}
 
-{% block modernizr %}
-    <!-- Custom modernizr for touch detection and svg support only, see the .js file for the build link -->
-    <script src="{{ static('nemo/_/js/modernizr.custom.js') }}"></script>
```

Comment:
Hmmm I'm pretty sure things will break if we remove this. The modernizr used by the main Wagtail templates does not include all of the features the Spanish site used - here's what the Spanish site does: http://modernizr.com/download/#-backgroundsize-svg-touch-cssclasses-addtest-prefixed-teststyles-testprop-testallprops-hasevent-prefixes-domprefixes

Reply:
What's the "touch detection" settings set there? I only see "SVG" and "background-size" set in that link. Those two appear to be used for IE8. How're we handling SVG icons in the Spanish site?

Reply:
It looks like possibly we're not handling them well -- see enlarged icon on https://www.consumerfinance.gov/es/ right now.

Reply:
Looks like this was introduced by the new `parse_links` middleware. The icon is being automatically added to external links now, but it wasn't before that middleware was introduced. PR opened at #4510 to fix.